src/main/resources/templates/admin.html

```diff
@@ -19,27 +19,23 @@ <h1>Pending Sightings</h1>
                     There was an error loading the pending sightings.
                 </div>
 
-                <div th:if="${sightings==null and sightings.empty}">
+                <div th:if="${sightings.empty}">  <!-- THis need to be tested!!! -->
```

Comment:
Can you add a ticket number to this TODO, so that it is clear as part of which ticket this will be tested?

Reply:
The ticket is WS-072-AdminPage I will add the ticket number there